app/src/main/java/com/concordia/mcga/fragments/TransportButtonFragment.java

```diff
@@ -392,7 +409,7 @@ public Animation getTransport_textview_close() {
         return transport_textview_close;
     }
 
-    public Transportation getTransportType() {
+    public String getTransportType() {
```

Comment:
The external library uses strings instead of enum. Could be fixed by wrapping the external library class. Will fix.